angular/package.json

```diff
@@ -1,6 +1,23 @@
 {
   "name": "rncp",
   "version": "0.0.0",
+  "contributors": [
+	{
+		"name": "David Rodríguez",
+		"email": "darodrig@student.42madrid.com",
```

Comment:
I'd rather not show my email publicly to avoid scrapers and bots find my account and potentially send spam.